config/api_client.go

```diff
@@ -64,6 +66,20 @@ func (c *Config) NewApiClient() (*httpclient.ApiClient, error) {
 				*r = *r.WithContext(ctx) // replace request
 				return nil
 			},
+			func(r *http.Request) error {
+				// Detect if the SDK is being run in a Databricks Runtime.
+				v, ok := os.LookupEnv("DATABRICKS_RUNTIME_VERSION")
+				if !ok {
+					return nil
+				}
+				// Replace any slashes and spaces with dashes, just in case.
+				v = strings.ReplaceAll(v, "/", "-")
+				v = strings.ReplaceAll(v, " ", "-")
+				// Add the detected Databricks Runtime version to the user agent
+				ctx := useragent.InContext(r.Context(), "runtime", v)
```

Comment:
We have to call it "runtime" in this case because the Python SDK already calls it runtime, and we don't want to diverge from that convention.
see: https://github.com/databricks/databricks-sdk-py/pull/287

The string constant part is done!